PonyGE2/src/algorithm/parameters.py

```diff
@@ -7,6 +7,7 @@
 
 """Algorithm parameters"""
 params = {
+    'DYNAMIC_BNF': False,
```

Comment:
boolean that allows us to decide whether we want to create a fully dynamic grammar or run in the normal mode.